src/Image.js

```diff
@@ -0,0 +1,75 @@
+import Modal from 'react-responsive-modal';
+import 'react-responsive-modal/lib/react-responsive-modal.css';
+
+import { PropTypes as T } from 'prop-types';
+import React, { Component } from 'react';
+
+import { clarifaiKey, clarifaiModel } from './config';
+const Clarifai = require('clarifai');
+const clarifaiClient = new Clarifai.App({
+  apiKey: clarifaiKey,
+ });
+
+class Image extends Component {
+  static propTypes = {
+    imageKey: T.number.isRequired,
+    url: T.string.isRequired,
+  }
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      imageKey: props.imageKey,
+      url: props.url,
+      open: false,
+      details: [],
+    };
+  }
+
+  onOpenModal = () => {
+    const url = this.state.url
+
+    clarifaiClient.models.predict(clarifaiModel, [url])
+      .then((response, err) =>
+        { if (response.status.description === 'Ok') {
```

Comment:
if you're interested, you could look into proxies to handle this and all the difference potential descriptions.
otherwise ,at least move the opening bracket up one line 👍